cea/interfaces/dashboard/plots/routes.py

```diff
@@ -65,22 +71,28 @@ def route_new_dashboard():
     """
     cea_config = current_app.cea_config
     plot_cache = current_app.plot_cache
-    dashboard_index = cea.plots.new_dashboard(cea_config, plot_cache, request.form.get('name'), request.form.get('description'))
+    dashboard_index = cea.plots.new_dashboard(cea_config, plot_cache, request.form.get('name'),
+                                              request.form.get('description'), request.form.get('layout'))
     return redirect(url_for('plots_blueprint.route_dashboard', dashboard_index=dashboard_index))
 
 
-@blueprint.route('/dashboard/<int:dashboard_index>/<func>')
+@blueprint.route('/dashboard/<func>/<int:dashboard_index>')
 def route_manage_dashboards_function(dashboard_index, func):
```

Comment:
why is this one function and not 3 separate functions?

- `route_delete_dashboard`
- `route_rename_dashboard`
- `route_duplicate_dashboard`

Reply:
hmm.. I probably thought it was a good idea and didn't want to keep copying the same few lines of code.